MMOCoreORB/src/server/zone/managers/resource/resourcespawner/ResourceSpawner.cpp

```diff
@@ -301,28 +301,49 @@ bool ResourceSpawner::writeAllSpawnsToScript() {
 	try {
 
 		File* file = new File("scripts/managers/resource_manager_spawns.lua");
+		File* craft = new File("scripts/managers/intendedspawns.txt");
```

Comment:
I'd rename this to something like `resource_manager_swgcraft_spawns.txt` so this is more generic and usable by everyone if they fork or merge our code into theirs.

Same for the galaxy harvester one: `resource_manager_galaxyharvester_spawns.xml`